src/components/base/Chart/Tooltip.js

```diff
@@ -46,35 +30,7 @@ const Tooltip = ({
       }}
     >
       <Container style={{ textAlign: 'center' }}>
-        {renderTooltip ? (
-          renderTooltip(item)
-        ) : (
-          <Fragment>
-            <FormattedVal
-              color="dark"
-              fontSize={5}
-              alwaysShowSign={false}
-              showCode
-              unit={counterValue.units[0]}
-              val={item.value}
-            />
-            {unit && (
-              <FormattedVal
-                color="grey"
-                fontSize={3}
-                alwaysShowSign={false}
-                showCode
-                unit={unit}
-                val={item.originalValue}
-              />
-            )}
-            <Box ff="Open Sans|Regular" color="grey" fontSize={3} mt={2}>
-              {moment(item.date)
-                .add(1, 'second')
-                .format('LL')}
-            </Box>
-          </Fragment>
-        )}
+        {renderTooltip ? renderTooltip(item) : null}
```

Comment:
I guess if `renderTooltip` is undefined we can avoid render the whole Tooltip

Reply:
good point